app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionOverlay.kt

```diff
@@ -81,13 +89,18 @@ class TrackingProtectionOverlay(
         val xOffset = res.getDimensionPixelSize(R.dimen.tp_onboarding_x_offset)
 
         // Positioning the popup above the tp anchor.
-        val yOffset = -containerHeight - (toolbar.height / 3 * 2) + triangleHeight
+        val yOffset = if (isBottomToolbar) {
+            -containerHeight - (toolbar.height / 3 * 2) + triangleHeight
+        } else {
+            CFR_Y_OFFSET
+        }
 
         trackingOnboarding.showAsDropDown(trackingProtectionIcon, xOffset, yOffset)
         settings.incrementTrackingProtectionOnboardingCount()
     }
 
     private companion object {
+        private const val CFR_Y_OFFSET = -30
```

Comment:
Can you adjust this to `-24` or `-25` to prevent the triangle overlapping the grey area of the toolbar? Same like with the bottom version.

Reply:
Done! Matches bottom now